src/main/java/com/hbm/items/tool/ItemPipette.java

```diff
@@ -0,0 +1,175 @@
+package com.hbm.items.tool;
+
+import api.hbm.fluid.IFillableItem;
+
+import com.hbm.inventory.fluid.FluidType;
+import com.hbm.inventory.fluid.Fluids;
+import com.hbm.items.ModItems;
+
+import cpw.mods.fml.relauncher.Side;
+import cpw.mods.fml.relauncher.SideOnly;
+
+import net.minecraft.client.renderer.texture.IIconRegister;
+import net.minecraft.entity.player.EntityPlayer;
+import net.minecraft.item.Item;
+import net.minecraft.item.ItemStack;
+import net.minecraft.nbt.NBTTagCompound;
+import net.minecraft.util.ChatComponentText;
+import net.minecraft.util.IIcon;
+import net.minecraft.world.World;
+
+import java.util.List;
+
+public class ItemPipette extends Item implements IFillableItem {
+
+    public ItemPipette() {
+        this.setMaxDamage(0);
+    }
+
+    @SideOnly(Side.CLIENT) protected IIcon overlayIcon;
+
+    public int amount = this.getMaxFill();
+
+    public FluidType type = Fluids.NONE;
+
+    public int getMaxFill() {
+        if(this == ModItems.pipette_laboratory)
+            return 50;
+        else
+            return 1_000;
+    }
+
+    public void initNBT(ItemStack stack) {
+
+        stack.stackTagCompound = new NBTTagCompound();
+
+        this.setFill(stack, type, 0);
+    }
+
+    public void setFill(ItemStack stack, FluidType type, int fill) {
+        if(!stack.hasTagCompound()) {
+            initNBT(stack);
+        }
+
+        this.type = type;
+        stack.stackTagCompound.setInteger(type.getName(), fill);
+    }
+
+    public int getFill(ItemStack stack, FluidType type) {
+        if(!stack.hasTagCompound()) {
+            initNBT(stack);
+        }
+
+        return stack.stackTagCompound.getInteger(type.getName());
+    }
+
+    @Override
+    @SideOnly(Side.CLIENT)
```

Comment:
pretty sure that annotation will at best not let this work on servers and at worst crash the game